### ISSUE TITLE
Can we add Committer Members on the members page

### ISSUE THREAD

**Issue by TanjaObradovic** [jakartaee/jakarta.ee#763]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by edbratt** [comment]
Is this a request to produce a list of committer members to each EE4J Project, qualified by the committer member organizations that listed on the membership/members page? (i.e. for Red Hat, list all contributors with committer roles in EE4J ... and do the same for all organizations listed?)

**Comment by TanjaObradovic** [comment]
This is a request to find a way to list all committer members including both 1)committers employed by Jakarta EE member organizations as well as 2) individual committer members not employed by a Jakarta EE member organization.

**Comment by edbratt** [comment]
So, this would be the uniqued list of everyone listed on all the EE4J project "Who is involved" pages (minus the mentors, I guess). Correct? That number is approaching 200 names. Seems like a large list of names. I don't have an objection but I do wonder, what value this will serve. This data is already available to anyone sufficiently diligent to click through those user page tables. Due to its size, I don't know if I would put it directly on the members page. Maybe a sub-page, or something requiring a click to expand?

**Comment by ivargrimstad** [comment]
How about displaying the **Individual** Committer Members? That is, those that are **not** employed by a member company but have signed the paperwork as individuals.

**Comment by shabnammayel** [comment]
Maybe add a "Committer Members" button or "Individual Committer Members" button in the banner which goes to a page that features them? I think we should keep the member organizations list and page separate from Committer members list

**Comment by TanjaObradovic** [comment]
Thanks all for the discussion, we want to find an optimal and reasonable way of exposing the information. Currently I am thinking we should have a section for Committer Members and list number of them, and possibly a button or a link that will lead to all Committer Members names.

**Comment by edbratt** [comment]
If this is valuable, it may be valuable to other working groups. Perhaps also consider if this would make sense at the working group home-page (e.g.https://projects.eclipse.org/projects/ee4j). Generating this list manually would be lots of work and perpetually out of date. I hope that isn't being considered.

**Comment by aeiras** [comment]
Hola everyone, 

@edbratt you hit the nail on the need to make the space be populated via Git and feed to the page automatic. 

Allow me to interject in this ticket and use MicroProfile Contributors' page as an example:
[MP Contributors Page](https://microprofile.io/contributors/)

**Value**

- automatic updates using Git, the tool of the project is necessary. If the page requires individual managing, it ought to not exist as there is no value on such waist of time

- the page welcomes anyone via actions to be listed as a Contributor. For MP, we choose not to separate Committers & Contributors. We understand that both are relevant and equally important to the project ecosystem. 

- Tha landing page has a Call to Action that brings you to the main adoc that tells anyone how to become involved and welcomes anyone to add via PRs more stuff to the list of how. 

- Using Git tooling to populate the page means that the contributors are in GIT, this is the most valuable stuff b/c each pic under MP contributor if you click it it tells you how that contributor is via its activity in git. Exactly what we need. 

Lastly, I recommend that: 

1. a new Tab is created to focus on Contributors or Committers (lets ask the community what they prefer by informal doodle poll)  
Mindful that Contributors can be anyone not only coders, the minimum requirement is to help via git, etc. With translations activities happening right now, we know that code is just 1 way to contributed. There are speakers, writers, designers, branding, etc. All valuable. 

To earn commit status is still a vacuum under the Jakarta EE project, I lean towards acknowledging anyone who is proactively moving the project forward instead of just focusing on the Committers, who are 165 today. -

**Comment by TanjaObradovic** [comment]
@chrisguindon, yes, lets list them (for now) as you are suggesting and in the style of meeting minutes.

**Comment by aeiras** [comment]
> This would require some changes to our Project API.
> 
> With this said, here's what we can do soon with a bit of work:
> 
> Each time we deploy an update to jakarta.ee, we fetch metadata for each Jakarta EE project from our API's.
> 
> This data includes a list of committers for each project. I can use this information to create an HTML list of all the unique committers for Jakarta EE. We could also add a link to the Eclipse Profile page of each committer.
> 
> For example:
> 
> * [Tanja Obradovic](https://accounts.eclipse.org/users/tobradovic)
> * [Christopher Guindon](https://accounts.eclipse.org/users/cguindon)
> * [Amelia Eiras](https://accounts.eclipse.org/users/aeiras)
> 
> This is the data that I can use now. Anything missing from this file will require some work to our APIs:
> 
> [eclipsefdn_projects.yaml.zip](https://github.com/jakartaee/jakarta.ee/files/4760645/eclipsefdn_projects.yaml.zip)

this sounds wonderful step forward @chrisguindon, thank you and everyone involved in this task! :) 

Chris, i adore your example. Currently, the Jakarta EE project doesn't recognize business contributions as part of earning committer status. Lets see if we can change that in the future by its community. Thats the reason TT has not assigned anyone in the business side of the project, currently active, to a committer.   Cheers!

**Comment by edbratt** [comment]
Committer lists don't change much. I'm not sure you need to do this for 
every new deployment. If it's not time-consuming, probably that doesn't 
matter. But if this query is lengthy, maybe updating this once a week or 
perhaps even monthly would suffice.

**Comment by aeiras** [comment]
agreed @edbratt.  But Chris hasn't told us how this temporary fix could be sustain until the API is a reality. Lets see what his recommendation is. I am very against taking human time from tools that ought to provide that data instead of human managing. Yet I do like the solution for now. :) it is an improvement!

**Comment by chrisguindon** [comment]
I am sorry if I my comment is causing some confusion. 

My suggestion is an automated process. 

We will be using public data that's currently available in our API. I just need to make a small change to the API to include the full name of each committee and we will be able to implement the list of committers.

**Comment by autumnfound** [comment]
@TanjaObradovic is the idea with the list to display each projects' committers separately or to have a large flat list of every unique committer?

**Comment by TanjaObradovic** [comment]
@autumnfound the idea is to have flat list of every unique committer.
